src/styles/modal.css

```diff
@@ -0,0 +1,108 @@
+/* modal */
+
+.ui.modal {
+  background-color: var(--shade-white);
+  border-radius: 0;
+  box-shadow: var(--box-shadow-5);
+}
+
+.ui.modal > .icon:first-child + * {
+  border-radius: 0;
+}
+
+/* close icon */
+
+.ui.modal > .close {
+  color: var(--shade-white);
+  font-size: 1.5rem;
+  height: 1.5rem;
+  padding: 0;
+  right: var(--spacing-2);
+  top: var(--spacing-3);
+  transition: opacity var(--transition-hover);
+  width: 1.5rem;
+}
+
+@media only screen and (max-width: 991px) {
+  .ui.modal > .close {
+    color: var(--shade-white);
+    font-size: 1.5rem;
+    height: 1.5rem;
+    padding: 0;
+    right: var(--spacing-2);
+    top: var(--spacing-3);
+    transition: opacity var(--transition-hover);
+    width: 1.5rem;
+  }
+}
+
+@media only screen and (max-width: 767px) {
+  .ui.modal > .close {
+    color: var(--shade-white);
+    font-size: 1.5rem;
+    height: 1.5rem;
+    padding: var(--spacing-2) 0;
+    right: var(--spacing-2);
+    top: var(--spacing-3);
+    transition: opacity var(--transition-hover);
+    width: 1.5rem;
+  }
+
+  .ui.modal > .actions {
+    margin-bottom: var(--spacing-3);
+  }
+
+  .ui.modal .content > .description {
```

Comment:
Let's create separate media queries for these selectors. I know it's a bit redundant but we basically want all styles for one particular selector together in one spot, so we don't need to scroll the file.

```
// all description styles
.description {
  // styles
}

@media screen... {
  .description {
    // styles
  }
}

// all something styles
.something {
  // styles
}

@media screen... {
  .something {
    // styles
  }
}